.github/workflows/publish-rpm-package.yml

```diff
@@ -80,11 +81,15 @@ jobs:
           HZ_LICENSEKEY: ${{ secrets.HZ_LICENSEKEY }}
         run: |
           source ./common.sh
+
+          # Bake authentication into the returned URLs
           wget \
             ${RPM_REPO_BASE_URL}/${PACKAGE_REPO}/hazelcast-rpm-${PACKAGE_REPO}.repo \
             --header "Authorization: Bearer ${{ env.JFROG_TOKEN }}" \
-            --output-document hazelcast-rpm-${PACKAGE_REPO}.repo
-          mv hazelcast-rpm-${PACKAGE_REPO}.repo /etc/yum.repos.d/          
+            --output-document - | \
+          sed "s#\(https://\)#\1${{ secrets.JFROG_USERNAME }}:${{ secrets.JFROG_TOKEN }}@#g" > \
```

Comment:
Use env vars instead `${{ env.JFROG_USERNAME}}`  and  `${{ env.JFROG_TOKEN }}`?

Otherwise no need for `JFROG_USERNAME: ${{ secrets.JFROG_USERNAME }}`

Reply:
> Otherwise no need for `JFROG_USERNAME: ${{ secrets.JFROG_USERNAME }}`

It's referenced in the `common.sh` so required for that.

Reply:
Ok.

Can we change both to use `env.`  for consistency
Couple of lines above and in other places  `${{ env.JFROG_TOKEN }` is used

Reply:
955c36ca18079cd8282309887d9f7b4381773fa7